package.json

```diff
@@ -5,7 +5,7 @@
   "main": "dist/glamorous.cjs.js",
   "jsnext:main": "dist/glamorous.es.js",
   "module": "dist/glamorous.es.js",
-  "typings": "typings/glamorous.d.ts",
+  "typings": "dist/glamorous.d.ts",
```

Comment:
I preferred to do it this way rather than just publish the `typings` directory because I like the clean separation between the source code and the published code.